examples/QuickJournalSync/Messages/EntryModifiedMessage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using CommunityToolkit.Mvvm.Messaging.Messages;
+using QuickJournalSync.Models;
+
+namespace QuickJournalSync.Messages
```

Comment:
Can move this to file-scoped namespace (also in the other files as well)

```suggestion
namespace QuickJournalSync.Messages;
```

Reply:
Done